pkg/sriov/sriov.go

```diff
@@ -226,3 +239,32 @@ func ReleaseVF(args *skel.CmdArgs) error {
 	})
 
 }
+
+// ResetVF reset the VF which accidently moved into default network namespace by a container failure
+func ResetVF(args *skel.CmdArgs, deviceID string) error {
+	hostIFName, cRefPath, err := LoadHostIFNameFromCache(args)
+	if err != nil {
+		return err
+	}
+	// get smart VF netdevice from PCI
+	vfNetdevices, err := sriovnet.GetNetDevicesFromPci(deviceID)
+	if err != nil {
+		return err
+	}
+	// Make sure we have 1 netdevice per pci address
+	if len(vfNetdevices) != 1 {
+		// This would happen if netdevice is not yet visible in default network namespace.
+		// so return ErrLinkNotFound error so that multus can attempt multiple times
```

Comment:
Is this behaviour specific for multus only?

Reply:
this can happen with any meta plugins that use ovs-cni, but I've tested only with multus. now changed it into generic term meta plugin.

Reply:
Thank you